inc/PluginSettings/PluginSettings.php

```diff
@@ -154,19 +155,36 @@ public static function encrypt_option( array $new_value, string|array|bool $old_
 		}
 	}
 
-	public static function decrypt_option( string $value ): array|null {
+	public static function decrypt_option( string $value ): array {
 		$decryptor = new \RemoteDataBlocks\WpdbStorage\DataEncryption();
+		$is_error = false;
 
 		try {
 			$decrypted = $decryptor->decrypt( $value );
-			return json_decode( $decrypted, true );
+
+			if ( false === $decrypted ) {
```

Comment:
This fixes the original stack trace in the issue description. `$decryptor->decrypt()` may return `false`, which causes the original error when `json_decode()` is called:

```
PHP Fatal error:  Uncaught TypeError: json_decode(): Argument #1 ($json) must be of type string, bool given in /var/www/html/wp-content/plugins/remote-data-blocks/inc/PluginSettings/PluginSettings.php:162
Stack trace:
#0 /var/www/html/wp-content/plugins/remote-data-blocks/inc/PluginSettings/PluginSettings.php(162): json_decode(false, true)
```

Reply:
From testing, I think the reason this fails is that `LOGGED_IN_KEY` and `LOGGED_IN_SALT` have different values during `wp-env` initial setup and once the site is actually running. I tried adding `error_log()`s [in the relevant code](https://github.com/Automattic/remote-data-blocks/blob/d73c623aa6124e86af2e8f82e93187bbb350e4da/inc/WpdbStorage/DataEncryption.php#L114) where we use these values and both values changed during initialization of `wp-env` after a version change and during normal website usage.

Because the values are initially encrypted with the site's real `LOGGED_IN_` keys, this process only fails during `wp-env` initialization.